docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -373,7 +373,7 @@ referenced by the `docker-compose.yml` file.
 
 ["source","txt",subs="attributes"]
```

Comment:
I wonder if we should take out the `subs="attributes"` since I think this is what prompted the change in #1851 (since the file on its own on GitHub doesn't work without the Asciidoc templating to replace variables

Reply:
Nevermind. I think we can do this for the `docker-compose.yml` file, but not the `.env` file since it still includes `{version}`.